Tests/Benchmarks/Benchmarks.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net471</TargetFramework>
+    <TargetFramework>net47</TargetFramework>
```

Comment:
Why is this needed?

Reply:
This is not needed, but I would recommend this for consistency of the solution.
The target net47 is also build for 4.7.0.
Currently you need to install SDK for 4.7.0 **and** 4.7.1 to build the solution.